src/Model/TalkCommentMapper.php

```diff
@@ -235,6 +236,57 @@ protected function getBasicSQL($include_hidden = false)
         return $sql;
     }
 
+    /**
+     * @param int $user_id
+     * @throws RateLimitExceededException
+     */
+    public function checkRateLimit(int $user_id)
+    {
+        if (isset($this->_request)) {
+            $max_comments = (int) $this->_request->getConfigValue('talk_comments_rate_limit_limit', 30);
+            $duration_minutes = (int) $this->_request->getConfigValue('talk_comments_rate_limit_reset', 60);
+        } else {
```

Comment:
I'd rather see this else removed by initializing the variable with the default values from the `else` block and only in case there is a request object set the variables default value is modified.